content/docs/web-components.md

```diff
@@ -6,9 +6,9 @@ redirect_from:
   - "docs/webcomponents.html"
 ---
 
-React y [Web Components](https://developer.mozilla.org/es/docs/Web/Web_Components) estan construidos para resolver diferentes problemas.  `Web Components` proporciona una fuerte encapsulación para componentes reutilizables, mientras que React proporciona una biblioteca declarativa que mantiene el DOM sincronizado con sus datos. Los dos objetivos se complementan. Como desarrollador, eres libre de usar React en tus `Web Components`, utilizar `Web Components` en React, o ambos.
+React y [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) están construidos para resolver diferentes problemas.  `Web Components` proporciona una fuerte encapsulación para componentes reutilizables, mientras que React proporciona una biblioteca declarativa que mantiene el DOM sincronizado con tus datos. Los dos objetivos se complementan. Como desarrollador, eres libre de usar React en tus `Web Components`, utilizar `Web Components` en React, o ambos.
```

Comment:
Thanks @Paolore 
@carburo pointed out that we are using the link to the MDN or Wikipedia in Spanish, so your original link to [https://developer.mozilla.org/es/docs/Web/Web_Components](https://developer.mozilla.org/es/docs/Web/Web_Components) was right. Please change back, sorry.

Reply:
Ok no problem